src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1554724063.inc.php

```diff
@@ -0,0 +1,76 @@
+<h1>Build #1554724063</h1>
+<h2>Date: 2019-04-08</h2>
+<div class="changelog">
+    - Remove workflow and revision management roles.
+</div>
+<?php
+
+use Doctrine\Common\Collections\Expr\Comparison;
+
+$databaseConnection = TCMSLogChange::getDatabaseConnection();
+
+$rolesRaw = $databaseConnection->fetchAll("SELECT `id` FROM `cms_role` WHERE `name` IN ('cms_revision_author', 'workflow_publisher', 'workflow_user')");
+$roles = [];
+foreach ($rolesRaw as $roleRow) {
+    $roles[] = $roleRow['id'];
+}
+
+if (0 === \count($roles)) {
+    return;
+}
+
+$data = TCMSLogChange::createMigrationQueryData('cms_role_cms_role_mlt', 'en')
+    ->setWhereExpressions([
+        new Comparison('source_id', Comparison::IN, $roles)
+    ])
+;
+TCMSLogChange::delete(__LINE__, $data);
```

Comment:
Shouldn't there be some sort of fallback (group)?

When simply removing the connections this could resolve in strange effects  - ie a backend menu point not being shown?